Source/f_finale.c

```diff
@@ -302,10 +372,19 @@ void F_TextWrite (void)
   int         cx;
   int         cy;
   
+  // [FG] if interbackdrop does not specify a valid flat, draw it as a patch instead
```

Comment:
This code looks familiar. So, we are supporting Rev 1.3 of the spec. Could you please add the UMAPINFO document from prboom-plus to the docs, with an install rule?